Sources/Kitura/Router.swift

```diff
@@ -157,6 +160,46 @@ public class Router {
         self.all(route, middleware: subrouter)
         return subrouter
     }
+
+    /// Handle an URL parameter
```

Comment:
Can these comments please explain WHY someone would use this functionality? What is the purpose of the feature / what problem does it solve?